pynicotine/gtkgui/userbrowse.py

```diff
@@ -77,7 +77,13 @@ def on_switch_browse_page(self, notebook, page, page_num):
 
         for tab in self.pages.values():
             if tab.Main == page:
-                GLib.idle_add(lambda: tab.FolderTreeView.grab_focus() == -1)
+
+                # Remember folder or file selection
```

Comment:
Fixes a bug where the focus was forgotten when switching tabs, and any previously file selection was lost when navigating with the keyboard (up and down cursor keys).